templates/load_initial_data.py

```diff
@@ -29,3 +36,50 @@
         site.domain = '{{ inventory_hostname }}'
         site.save()
         print('default site updated')
+
+# Read SSH key pair
+with open('/home/{{www_user}}/.ssh/id_rsa', mode='r') as file:
+    ssh_private_key  = file.read()
+with open('/home/{{www_user}}/.ssh/id_rsa.pub', mode='r') as file:
+    ssh_pub_key  = file.read()
+
+# Create a default credentials object
+try:
+    Credentials.objects.create(
```

Comment:
I think it's safe to create it only if no other credentials exist, the `IntegrityError` is a bit undesired since it generates noise in the postgres logs